code/modules/mob/living/carbon/human/life.dm

```diff
@@ -812,6 +827,38 @@
 		bodytemperature += recovery_amt
 
 	//This proc returns a number made up of the flags for body parts which you are protected on. (such as HEAD, UPPER_TORSO, LOWER_TORSO, etc. See setup.dm for the full list)
+/mob/living/carbon/human/proc/add_hypothermia(var/amount)
+	if(hypothermia+amount<=0)
+		hypothermia=0
+	else
+		if(hypothermia+amount>100)
+			hypothermia=100
+		hypothermia+=amount
+		if(hypothermia>5 && hypothermia<75)
+			if(prob(10))
+				say("*shiver")
+		if(hypothermia>30)
+			if(prob(50))
+				stuttering = hypothermia-25
+			if(hypothermia>50)
+				if(prob(40))
+					apply_effect(hypothermia-45,EYE_BLUR)
+				if(hypothermia>70)

```

Comment:
all of these nested if statements should be broken apart into distinct clauses. so that they can use the `if(hypothermia > threshold && prob)` syntax outlined above.